### PR TITLE
TEP007: Adding Isotope support

### PR DESCRIPTION
**Pull request by wkerzendorf**

### CODE REVIEW

File: TEP007_isotopes_decay.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Backward compatibility
+======================
+
+This TEP proposes an addon to the code and does not change backwards
+compatibility.
+
+Alternatives
+============
+
+Currently None
```
Comment (unoebauer) [inline TEP007_isotopes_decay.rst:54]: Do the decay offline - that's how we are currently doing it in Talytha's project

File: TEP007_isotopes_decay.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+specific isotopes and with hardcoded numbers. In addition, the energy released
+by the decay could be calculated and used in the TARDIS simulation which
+currently it does not do.
+
+We propose to include a method to specify and decay isotopes within the TARDIS
+framework. This will include the internal code structure as well as the means to
+specify those within the configuration file. Thus this TEP has some links to
+the proposed changes in TEP006 as an implementation of TEP006 will make this
+TEP easier.
+
```
Comment (unoebauer) [inline TEP007_isotopes_decay.rst:34]: Is the idea here to be generic or really to be restricted to supernova-relevant decay chains?

**Comment by unoebauer** [comment]
In principle a very good idea - and urgently needed. But maybe we should become a little more concrete, in particular concerning the scope of this decay handler (only Ni chain?).

File: TEP007_isotopes_decay.rst (hunk 2 above)
Comment (wkerzendorf) [inline TEP007_isotopes_decay.rst:34]: @unoebauer the idea is to be generic - the reason being that we can just use `pyne` that does that automatically. Essentially we are not implementing the chains, we are only implementing the framework that does that. I've used it in tardisnuclear and it works very well.

File: TEP007_isotopes_decay.rst (hunk 1 above)
Comment (wkerzendorf) [inline TEP007_isotopes_decay.rst:54]: Will add this.

**Comment by wkerzendorf** [comment]
@unoebauer - if there are no further suggestions or questions -  I let you merge. If there are - let me know

**Comment by wkerzendorf** [comment]
@unoebauer in terms of concrete - look at the mentioned PR.

**Pull request merged**